thrifty-integration-tests/bar/Bar.thrift

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Actually, per my other comment, this and `Foo.thrift` should go away, and the test moved to a junit case in `LoaderTest.java`